### DISCUSSION TITLE
Help my very important 3 year old github profile got suspended with all my importatn stuff help

### DISCUSSION THREAD

**Discussion by RahulPatel27** [community/community#123505] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i dont know i just had commented on the community on discussion and certainly my main profile got suspended i dont know why and i dont know what to do i need it back really please help i beg you

**Comment by RahulPatel27** [comment]
and i did not get any mail why so please tell me what i should do

**Comment by ebndev** [comment]
Hi @RahulPatel27,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
